sa/storage-authority.go

```diff
@@ -636,3 +636,19 @@ func (ssa *SQLStorageAuthority) AlreadyDeniedCSR(names []string) (already bool,
 
 	return
 }
+
+// CountCertificatesRange returns the number of certificates issued in a specific
+// date range
+func (ssa *SQLStorageAuthority) CountCertificatesRange(start, end time.Time) (count int64, err error) {
+	err = ssa.dbMap.SelectOne(
+		&count,
+		`SELECT COUNT(*) FROM certificates
+		WHERE issued >= :windowLeft
+		AND issued <= :windowRight`,
```

Comment:
Inclusive on both sides will cause duplicate returns. Pick one to be exclusive. (Typically, that's the `<`)